### PR TITLE
add mock-data-gen to list

### PR DESCRIPTION
**Pull request by kaeluka**
Hey, I have been working on a library that maps `io-ts` codecs to `fast-check` arbitraries. The library that you already have on the list does not have support for a couple of things that I find indispensible in practical use (like generating arbitraries for intersection types and branded types).

### CODE REVIEW

**Comment by kaeluka** [comment]
Thanks for `fast-check` it's really enjoyable to work with :)

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@
 
 - [ava-fast-check](https://github.com/dubzzz/ava-fast-check) - Property based testing for AVA based on fast-check
 - [fast-check-io-ts](https://github.com/giogonzo/fast-check-io-ts) - io-ts codec to fast-check arbitrary mapping
+- [mock-data-gen](https://github.com/kaeluka/mock-data-gen) - Map io-ts to fast-check arbitraries with flexible support for extending generated data.
```
Comment (dubzzz) [inline README.md]: Please follow alphabetical order

**Review by dubzzz — COMMENTED** [review]
Thanks for the suggestion, just one minor remark before we merge it

File: README.md (hunk 1 above)
Comment (kaeluka) [reply README.md]: I've amended the PR (and also moved one other line to be in alphabetical order) :+1:

**Comment by kaeluka** [comment]
Done! Also moved one other line to be in alphabetical order (or was that ordered like that intentionally?)

**Review by dubzzz — APPROVED** [review]

**Pull request merged**